main/acle.md

```diff
@@ -8959,14 +8960,14 @@ Gather Load Quadword.
    // _bf16, _f16, _f32, _f64
    svint8_t svld1q_gather[_u64base]_s8(svbool_t pg, svuint64_t zn);
    svint8_t svld1q_gather[_u64base]_offset_s8(svbool_t pg, svuint64_t zn, int64_t offset);
-   svint8_t svld1q_gather_[u64]offset[_s8](svbool_t pg, const int8_t *base, svuint64_t offset);
+   svint8_t svld1q_gather[_u64]_offset[_s8](svbool_t pg, const int8_t *base, svuint64_t offset);
```

Comment:
Why are this being changed for offset. AFAIU this should follow the _u64base. So it means that the base is 64 bits. That is the reason it was [_u64]offset. So the offset is 64 bits

Reply:
Oh, I see, my mistake, I'll fix it.

Reply:
Done